docs/patterns/components/TwoColumnPanel/index.js

```diff
@@ -0,0 +1,361 @@
+import React, { useRef, useState } from 'react';
```

Comment:
Not sure if it's just me, but when I toggle back and forth a few times, the page gets slower and slower until it crashes.

https://user-images.githubusercontent.com/6416791/149158736-2c458c83-2dc9-40dc-830f-3bfc9bdf0ed9.mov

Reply:
I need to investigate this, I think the tooltip (popper.js) has some performance issues in the past so ideally we need to resolve this.

Reply:
Found the issue, will be addressed with #505